spec/forms/trainee_form_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+class TestForm < TraineeForm
+  attr_accessor :test1, :test2, :test3
+  validates_presence_of :test1, :test2, :test3
+
+private
+
+  def compute_fields
```

Comment:
Wondering what this is...

Reply:
TraineeForm needs that method implemented